### ISSUE TITLE
RuntimeError: unknown file type

### ISSUE THREAD

**Issue by jcoyne** [PerfectlyNormal/tinymce-rails-imageupload#87]
```
RuntimeError: unknown file type: /opt/rubies/ruby-2.3.1/lib/ruby/gems/2.3.0/gems/tinymce-rails-imageupload-4.0.17.beta.2/vendor/assets/javascripts/tinymce

    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:1387 :in `copy`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:472 :in `block in copy_entry`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:1498 :in `wrap_traverse`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:469 :in `copy_entry`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:444 :in `block in cp_r`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:1571 :in `block in fu_each_src_dest`

    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:1585 :in `fu_each_src_dest0`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:1569 :in `fu_each_src_dest`
    /opt/rubies/ruby-2.3.1/lib/ruby/2.3.0/fileutils.rb:443 :in `cp_r`
    [GEM_ROOT]/gems/tinymce-rails-4.5.6/lib/tinymce/rails/asset_installer/copy.rb:30 :in `copy_assets`
    [GEM_ROOT]/gems/tinymce-rails-4.5.6/lib/tinymce/rails/asset_installer/copy.rb:13 :in `call`
    [GEM_ROOT]/gems/tinymce-rails-4.5.6/lib/tinymce/rails/asset_installer.rb:22 :in `install`
    [GEM_ROOT]/gems/tinymce-rails-imageupload-4.0.17.beta.2/lib/tasks/tinymce-uploadimage-assets.rake:15 :in `block in <top (required)>` 
```

**Comment by dboyd68** [comment]
also noticed this issue

**Comment by PerfectlyNormal** [comment]
I have no idea what causes this, and not really the time to do a lot of debugging either.
I don’t use tinymce on any projects any longer, so if anybody has any ideas, pull requests welcome

**Comment by revgum** [comment]
@dboyd68 you might try the path fix like I have in PR #88 to verify

**Comment by botimer** [comment]
Yup -- pull request #88 should take care of this and clear up a number of these open issues.
